manifest.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 applications:
   - name: locust
-    no-route: true
+    routes:
+    - route: identity-idva-monitoring-locust-((ENVIRONMENT)).apps.internal
```

Comment:
Confused by this. Why does our load testing app need a route assigned to it? No other apps should ever be trying to send traffic here.

Reply:
in order to be able to access the ui over https

Reply:
Where is the access to this application coming from? Is this somewhere that we're setting up an SSH tunnel for and hitting locally? Or is another app going to be hitting `https://identity-idva-monitoring-localhost-prod.apps.internal:61443` from somewhere? I don't see the value in adding a route if we're using SSH to hit the app, but if it's another cf-app then I don't think I understand how we're utilizing this app.

Reply:
This route is only to allow hitting the app over https after opening an ssh tunnel.